app/controllers/suggestion_controller.rb

```diff
@@ -73,8 +73,8 @@ def create
     end
 
     if @suggestion.save
-      flash[:success] = 'Thank you for your suggestion!' if @suggestion.unityID != ''
-      flash[:success] = 'You have submitted an anonymous suggestion. It will not show in the suggested topic table below.' if @suggestion.unityID == ''
+      flash[:success] = 'Thank you for your suggestion!' unless @suggestion.unityID.empty?  #Change these two lines of codes to Ruby style
```

Comment:
Maybe just this one comment removal, rest looks good

```suggestion
      flash[:success] = 'Thank you for your suggestion!' unless @suggestion.unityID.empty? 
```